stdlib/sql/src/main/ballerina/src/sql/types.bal

```diff
@@ -0,0 +1,48 @@
+import ballerina/config;
+import ballerinax/java;
+
+# Represents the properties which are used to configure DB connection pool.
+# Default values of the fields can be set through the configuration API.
+#
+# + maxOpenConnections - The maximum number of open connections that the pool is allowed to have, including both idle and in-use connections.
+#                     Default value is 15 and it can be changed through the configuration API with the key
+#                     `b7a.sql.pool.maximumPoolSize`.
```

Comment:
The key here (`b7a.sql.pool.maximumPoolSize`) seems doesn't match with the actual key used (`b7a.sql.pool.maxOpenConnection`).

Reply:
Yes, I missed to rename the API key in the comment when I was changing the name. I'll make the necessary changes.